.gitignore

```diff
@@ -6,3 +6,4 @@ src/backend/config.hpp
 build-aux/.flatpak-builder/
 build-aux/.pdfslicer/
 build-aux/pdfslicer-repo/
+.vscode
```

Comment:
I don't think this is related to this MR, nor it should be in the project code. (if vscode doesn't hide it by default, it shouldn't be every single projects that needs to add this line)

Reply:
I didn't approve it explicitly, but I'm ok to that being committed to the repo. It is indeed standard practice to add any IDE-specific files to .gitignore. [This is the .gitignore file for range-v3](https://github.com/ericniebler/range-v3/blob/master/.gitignore), for example.

Reply:
In my opinion, this file change doesn't relate directly to this MR. As I also know, it's a common practice to add IDE system files (and any other files which prevents you from convenient git usage) to .gitignore. 
I added this in current MR just to not create a separate MR for this single line. But, yes, it's not so neat as it can be.
Anyway, I'm not insist on this change. @junrrein, please reach a verdict.

Reply:
Hey @miltolstoy, I agree with you, this change can stay. But I had already commented saying so in this conversation and marked it as resolved.